### DISCUSSION TITLE
Add Server ID as a Variable for Game Server webhooks

### DISCUSSION THREAD

**Discussion by Buck-Irl** [gameserverapp/Platform#794] [Suggestions]
As the Title states, would it be possible to get a Variable Id added to http link,

Currently, if it worked, I would have to set up a separate task for each task, on each server.
If it worked like the player webhook, I could have the mod send server id with request, then one task would work across the cluster.

On Related note, Don't wanna say it on discord without your permission... but the mod id is 980023 
I got it made for GSA webhooks, handy for us console people;) Am i allowed tell the community about it? 
The Current Version has 2 or 3 visual bugs, although the webhooks are working perfectly. So I wont say a word yet either way. 
The bugs have been fixed, just need CF to approve it tomorrow or Monday and the mod will be complete.

**Comment by mvaessen** [comment]
this should be available now
